pkg/server/status.go

```diff
@@ -41,14 +42,16 @@ func (c *criService) Status(ctx context.Context, r *runtime.StatusRequest) (*run
 		Type:   runtime.NetworkReady,
 		Status: true,
 	}
+
+	// Load the latest cni configuration to be in sync with the latest network configuration
+	if err := c.netPlugin.Load(cni.WithLoNetwork(), cni.WithDefaultConf()); err != nil {
```

Comment:
seems like an expensive operation if there has been no change..?

Reply:
any of the moby repos using a fsnotify package?

Reply:
yes it reads the config file on every status check. I can add some intelligence in the Load on the go-cni side to only load if the checksum/date has changed. Keeping this simple and similar to kube upstream for now.

Reply:
yeah simple and low memory/fast